storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst

```diff
@@ -448,7 +446,7 @@ Command-line options
 
 .. option:: --swift-storage-url
 
-   xbcloud will try to get object-store URL for given region (if any specified)
+   xbcloud attempts to get object-store URL for given region (if any specified)
```

Comment:
xbcloud attempts to get an object-store URL for a specified region.

Reply:
Fixed.